docs/modules/ROOT/pages/running-commodore.adoc

```diff
@@ -45,7 +45,47 @@ commodore() {
 == macOS X
 
 On macOS with Docker for Mac mounting the SSH agents socket into a container doesn't work.
-Instead your SSH key needs to be mounted into the container.
+Instead you need to mount as magic path that's provided by Docker for Mac.
+
+[NOTE]
+====
+This only works for the stock ssh-agent coming along with macOS.
+If you use any other agent, you are out of luck.
+Docker for mac doesn't support mounting sockets.
+====
+
+[CAUTION]
+====
+That magic socket path belongs to root.
+One must a container with `--user=0` in order to access it.
+In consequence, the files created by Commodore will also belong to root.
+If you do want to edit them on the host, you need to run `chown -R $UID .` first.
+====
+
+[source,bash]
+----
+commodore() {
+  docker run \
+    --interactive=true \
+    --tty \
+    --rm \
+    --user="$(id -u)" \
+    --env COMMODORE_API_URL=$COMMODORE_API_URL \
+    --env COMMODORE_GLOBAL_GIT_BASE=$COMMODORE_GLOBAL_GIT_BASE \
+    --env COMMODORE_API_TOKEN=$COMMODORE_API_TOKEN \
+    --env SSH_AUTH_SOCK=/tmp/ssh_agent.sock \
+    --volume "/run/host-services/ssh-auth.sock:/tmp/ssh_agent.sock" \
```

Comment:
Doesn't OSX have `$SSH_AUTH_SOCK` which holds the path to the current ssh-auth socket?

Reply:
Yes, but the `/run/host-services/ssh-auth.sock` is a special path inside the Docker VM where the host's agent socket is mounted. So it's always the same path and the host's socket must be at a certain location.

Reply:
Ah right, there's one more layer of magic on OSX, I forgot that was the original issue. So, to reference the other comment, it should still work with any agent though, since that layer of mapping (host-level `$SSH_AUTH_SOCK` to `/run/host-services/ssh-auth.sock`) is done by Docker on OSX?

Reply:
Unfortunately Docker for mac maps a hard coded path into the VM (instead of the generic `$SSH_AUTH_SOCK`).

Reply:
well, nevermind then